test_env_vars.yaml

```diff
@@ -76,10 +76,17 @@ cached_dependencies:
     https_url: "https://github.com/release-engineering/retrodep.git"
   # Repository that will be used for the test
   test_repo:
+    # Use local bare remote repository
+    # If this is used, the git urls below should be paths and should be available for the workers
+    use_local: True
     # The tracked repository (remote)
-    ssh_url: "git@github.com:user/cachito_test_repo.git"
+    ssh_url: "./tmp/cachito-archives/integration-tests-git-repo.git"
```

Comment:
It seems that when running tests against the `docker-compose` deployment, you must manually create `./tmp/cachito-archives` prior to running `docker-compose up`.

Otherwise, docker automatically creates this directory as root and the test for cached dependencies fails with
```
>           bare_repo = git.Repo.init(str(bare_repo_dir), bare=True)
...
E           PermissionError: [Errno 13] Permission denied: '<cachito repo>/tmp/cachito-archives/integration-tests-git-repo.git'
```

Is this something that should be mentioned as a comment? Or is that general knowledge for running integration tests? Note that other tests work even if you let the directory be created automatically.

Reply:
Thanks for checking this!

I have not run through this issue because I am using podman to run the tests (and for now, I do have to manually create those directories).

We should either introduce changes to the Makefile to prepare the environment (creating the directories) and update the docs, or just drop this patch. Creating the directory was also part of #225, but should be deprecated by https://github.com/containers/podman-compose/pull/194 for that specific case.

Thoughts?

Reply:
What were the prerequisites for running this test before? Create any empty repo on github and specify it in the config file?

This does seem like a pretty nice quality-of-life change, so I wouldn't have anything against updating the Makefile and docs, but I'd let more experienced members weigh in on this.

Reply:
> What were the prerequisites for running this test before? Create any empty repo on github and specify it in the config file?

Yes.

> This does seem like a pretty nice quality-of-life change, so I wouldn't have anything against updating the Makefile and docs, but I'd let more experienced members weigh in on this.

Sounds good. Let's see what others think as well and then I will proceed to making these additional changes.

Reply:
Using the `Makefile` to perform setup actions prior to running `docker-compose up` sounds like a good idea.

Reply:
Done! I also added a comment in the Makefile to explain why we are doing it.